apps/prow/OWNERS

```diff
@@ -1,10 +1,15 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-required_reviewers:
-  - bentheelder
-  - cblecker
-  - spiffxp
+approvers:
+- ameukam
+- bentheelder
+- spiffxp
+
+reviewers:
+- ameukam
+- bentheelder
+- spiffxp
 
 labels:
-- area/prow/bump
 - sig/testing
+- area/apps/prow
```

Comment:
Formatting change: for apps, reorder so the "owning group" label is first, and the "app-specific" label is second